src/fontra/views/editor/editor.js

```diff
@@ -138,6 +138,22 @@ export class EditorController {
       }
     );
 
+    this.sceneSettingsController.addKeyListener("location", () => {
```

Comment:
This should be in panel-reference-font.js. The functionality should perhaps be a separate method, so that this can also be called when the reference font panel is initialized, not just when the sliders change.

Reply:
Additionally, this setting should also affect the reference glyph drawn in the canvas (see the visualization layer, marked with registerVisualizationLayerDefinition).

Reply:
It should be done already, in line 149.

Reply:
Same canvas is used for drawing the reference text. The only way to set variable glyph is doing by setting by css to the canvas element currently. There is no method for setting font variables in CanvasContext2D.

Reply:
Although I don't see any unexpected results in other text drawing, and indeed the cavas API doesn't seem to support `fontVariationSettings` yet. So let's go forwards with this.

Reply:
(Sorry, our messages crossed, I confirm what you wrote.)

Reply:
Then all that's left to do is move the code to the right module (first comment).

Reply:
Do you want me to separate the event callbacks into two, moving the canvas font setting to the `canvas-controller` and reference font related code to the `panel-reference-font`?

Reply:
I did.